client/my-sites/marketplace/components/review-item/index.tsx

```diff
@@ -38,7 +39,7 @@ export const MarketplaceReviewItem = ( props: MarketplaceReviewItemProps ) => {
 
 	const setEditing = ( review: MarketplaceReviewResponse ) => {
 		setIsEditing( true );
-		setEditorContent( review.content.raw );
+		setEditorContent( review.content.raw.replace( new RegExp( `^${ EMPTY_PLACEHOLDER }$` ), '' ) );
 		setEditorRating( review.meta.wpcom_marketplace_rating );
```

Comment:
I'm not sure if this regex is working, cause I can see it being rendered on the page.

![CleanShot 2024-01-17 at 11 58 43@2x](https://github.com/Automattic/wp-calypso/assets/5039531/81ed282b-74fa-49ec-84b5-dc530b209cd5)

Reply:
You can test that, it is replaced for the Edit mode, are you showing that HTML for the Edit textbox?

Reply:
Ah, sorry, it is showing on the display mode.